vignettes/sensors.Rmd

```diff
@@ -16,140 +16,179 @@ knitr::opts_chunk$set(
 ```
 
 ## Accelerometer
 
 #### Data format and Sample data
 
-`mhealthtools` already comes preloaded with sample accelerometer data. You can access the data through `data(accelerometer_data)`. The first five rows are shown below, to give you an idea of the structure of the data (this is the input format to the functions that extract accelerometer features). Where `t` is the timestamp; `x`,`y` and `z` are the X,Y and Z- axis respectively
+`mhealthtools` already comes preloaded with sample accelerometer data. You can access the data through `data(accelerometer_data)` or `mhealthtools::accelerometer_data`. The first five rows are shown below, to give you an idea of the structure of the data (this is the input format to the functions that extract accelerometer features). Where `t` is the timestamp; `x`,`y` and `z` are the X,Y and Z- axis respectively
 
-```{r data, echo = FALSE}
+```{r accel data, echo = FALSE}
 data(accelerometer_data)
 head(accelerometer_data,5)
 ```
 
-Let's plot to see how the data actually looks like 
+Let's plot to see how the accelerometer data actually looks like 
 
-```{r, echo=FALSE, fig.width=8, fig.height=4}
+```{r accel plots, echo=FALSE, fig.width=8, fig.height=4}
 library(ggplot2)
 a <- accelerometer_data
 a$t <- a$t - a$t[1]
 a <- tidyr::gather(a, 'axis' , 'val', -t)
 ggplot(a, aes(x = t, y= val)) + geom_line()+facet_wrap(~axis) + ylim(c(-0.025, 0.025))
 ```
+We are looking at 10s of data sampled at a high enough sampling rate(for accelerometer data) of 100Hz. The data looks noisy, and looks like it could make good use of some processing like frequency filters, maybe subsetting the time (time filtering) - all of which mhealthtools conveniently offers.
 
 #### Extracting features (using pre-existing pipeline)
-To use the existing feature set (you can define your own features or feed in your own model to extract features in `mhealthtools`!!). Note: You need to get the data in the proper format (as that of the sample data) to use the feature extraction pipeline.
+NOTE: To use the existing feature set (you can define your own features or feed in your own model to extract features in `mhealthtools`!!) you need to get the data in the proper format (as that of the sample data) to use the feature extraction pipeline.
 
-```{r, warning=FALSE, error=FALSE, prompt=FALSE}
+Let's take a look at the default feature extraction pipeline, whose features are all classical signal processing features based on time domain, frequency domain and the energy bands of the frequency spectrum.
+
+```{r accel default features, warning=FALSE, error=FALSE, prompt=FALSE}
 accel_features <- accelerometer_features(accelerometer_data)
 # using default parameters
 ```
+The output file is a list containing features extracted using supplied/default functions (extracted_features), features calculated using models(model_features) and an element for error (error). Let's look at the extracted_features to see how the default features look like.
```

Comment:
The reason I mentioned `extracted_features` was because that is the name of the key in the output list. I would explain the keys with their tag(in brackets), the whole sentence reads that way. I'll restructure it to be more coherent

Reply:
Ah, I see. I use a dollar sign `$extracted_features` elsewhere in the package to make it more clear that it's a named list element, but I haven't seen a convention around these things. At least format it as a code block so it is obvious we are talking about a code object.

Reply:
That's a good idea. I will move to `$extracted_features`, to be consistent across the package.